From 3a909d6076390ad2cc8fe33e8c41077281b87867 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C4=81rti=C5=86=C5=A1=20Ma=C4=8Ds?=
 <macs.martins@gmail.com>
Date: Fri, 11 Nov 2016 15:27:38 +0200
Subject: [PATCH 4/5] Revert "Close standard output after printing XID, patch
 due Abby Cedar."

This reverts commit 96041e5ba2e19a077cb8aead03899c0b1eef529a.

newwindow() needs stdout to write XID of the new windows.
---
 surf.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/surf.c b/surf.c
index 953a571..da1db7c 100644
--- a/surf.c
+++ b/surf.c
@@ -1111,8 +1111,6 @@ newclient(void)
 		printf("%u\n",
 		       (guint)GDK_WINDOW_XID(GTK_WIDGET(c->win)->window));
 		fflush(NULL);
-		if (fclose(stdout) != 0)
-			die("Error closing stdout");
 	}
 
 	return c;
-- 
2.7.3

